cmd/config/config.go

```diff
@@ -0,0 +1,97 @@
+package config
+
+import (
+  "os"
+  "strings"
+
+  "github.com/Originate/gt/cmd/util"
+)
+
+
+func GetMainBranch() string {
+  return getConfigurationValue("main-branch-name")
+}
+
+
+func GetParentBranch(branchName string) string {
+  return getConfigurationValue(branchName + ".parent")
+}
+
+
+func GetPerennialBranches() []string {
+  return strings.Split(getConfigurationValue("perennial-branch-names"), " ")
+}
+
+
+func GetPullBranchStrategy() string {
+  pullBranchStrategy := getConfigurationValueWithDefault("pull-branch-strategy", "rebase")
+  return pullBranchStrategy
+}
+
+
+func GetRemoteOriginUrl() string {
+  if os.Getenv("GIT_TOWN_ENV") == "test" {
+    mockRemoteUrl := getConfigurationValue("testing.remote-url")
+    if mockRemoteUrl != "" {
+      return mockRemoteUrl
+    }
+  }
+  return util.GetCommandOutput([]string{"git", "remote", "get-url", "origin"})
+}
+
+
+func GetRemoteUpstreamUrl() string {
+  return util.GetCommandOutput([]string{"git", "remote", "get-url", "upstream"})
+}
+
+
+func IsFeatureBranch(branchName string) bool {
+  return branchName != GetMainBranch() && !IsPerennialBranch(branchName)
+}
+
+
+func IsPerennialBranch(branchName string) bool {
+  perennialBranches := GetPerennialBranches()
+  return util.ContainsString(perennialBranches, branchName)
```

Comment:
I don't know if we are sorting them currently, but I like that idea and will switch to it and make sure to sort them when adding a new one (that hasn't been implemented yet)